README.md

```diff
@@ -90,6 +90,26 @@ An iPhone 7+ and iOS 11+ are required in order to use NFC with iOS devices.
 <string>com.apple.developer.nfc.readersession.iso7816.select-identifiers</string>
 ```
 
+#### iOS Conciderations
```

Comment:
Can you fix "Conciderations" by "Considerations"?

Reply:
Done!

Reply:
@franckbour any other observation? :)

Reply:
No, thansk!